compiler/mir/mirgen.nim

```diff
@@ -528,6 +525,29 @@ proc genRd(c: var TCtx, n: PNode; consume=false): Value =
     # either an rvalue or some non-pure lvalue -> capture
     result = captureInTemp(c, f, consume)
 
+proc capture(c: var TCtx, n: PNode): Value =
+  ## If not not a stable lvalue expression, captures the result of the
```

Comment:
```suggestion
  ## If not a stable lvalue expression, captures the result of the
```

IIUC, mutable (unstable) lvalues require capture, so the second `not` was a typo.

Reply:
Yep, it's a typo -- good catch.